solutions/observability/observability-ai-assistant.md

```diff
@@ -32,26 +32,18 @@ The {{obs-ai-assistant}} helps you:
 
 ## Requirements [obs-ai-requirements]
 
-The AI assistant requires the following:
+To set up or use AI assistant, you need the following:
 
-- An **Elastic deployment**:
+* An appropriate [Elastic license](https://www.elastic.co/subscriptions).
 
-  - For **{{observability}}**: {{stack}} version **8.9** or later, or an **{{observability}} serverless project**.
+* The `Observability AI Assistant: All` {{kib}} privilege.
 
-  - For **Search**: {{stack}}  version **8.16.0** or later, or **{{serverless-short}} {{es}} project**.
+* An [LLM connector](/solutions/security/ai/set-up-connectors-for-large-language-models-llm.md).
 
-    - To run {{obs-ai-assistant}} on a self-hosted Elastic stack, you need an [appropriate license](https://www.elastic.co/subscriptions).
-
-- An account with a third-party generative AI provider that preferably supports function calling. If your AI provider does not support function calling, you can configure [AI Assistant settings](../../solutions/observability/observability-ai-assistant.md#obs-ai-settings) to simulate function calling, but this might affect performance.
```

Comment:
I know function calling is something used in Observability that Security doesn't use. We may want to keep a note that recommends a model that supports function calling or to configure simulated function calling.

I would only think this wouldn't be necessary if all of the models we have connectors for support function calling. @viduni94 maybe you would have some insight into this?

Reply:
After my conversation with Viduni, I don't think we need to add a note. Recent proprietary models all support function calling, and if the selected model doesn't support function calling, we use simulated function calling so assistant still works.